addons/project/static/src/project_sharing/chatter/chatter_patch.js

```diff
@@ -13,10 +13,18 @@ patch(Chatter.prototype, {
         this.orm = useService("orm");
         useSubEnv({
             projectSharingId: this.props.projectSharingId,
-            inFrontendPortalChatter: true,
```

Comment:
Why was it set for? Is it safe to just remove it like that without compensating with condition on projectSharingId elsewhere in the code?

Reply:
It was for hiding the full composer button in project sharing (mentioned PR in the commit message #231360) which is now covered by [this change](https://github.com/odoo/odoo/pull/235551/files#diff-98a0d0a4f55cecb90026d9a4917b071d378b3df56ee7c9c892e22a6cda82e77aR6) in this PR.